README.md

```diff
@@ -54,7 +54,7 @@ Fast JSON API serialized 250 records in 3.01 ms
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'fast_jsonapi'
+gem 'fast_jsonapi', git: 'https://github.com/fast-jsonapi/fast_jsonapi'
```

Comment:
@nratter we should include the `tag: '1.6.0'` along, this will help with pinning the version.